src/icons/fileIcons.ts

```diff
@@ -2570,5 +2570,10 @@ export const fileIcons: FileIcons = {
         '.k8s.yaml',
       ],
     },
+    {
+      name: 'interface',
+      fileNames: ['interface.ts', 'interfaces.ts'],
+      fileExtensions: ['.ts'],
```

Comment:
This won't work anyways, because it would override all TypeScript files and would show your icon on all files.